packaging/debian/changelog

```diff
@@ -0,0 +1,5 @@
+azure-nvme-utils (99.99.99) UNRELEASED; urgency=low
```

Comment:
I'd use a version like 0.0~1, to ensure that it's always sorted as an earlier version than what may be available otherwise.  We don't want people accidentally upgrading to dev versions (it can still be installed if people really mean it)

Reply:
i switched the specs to using a dynamic version determined from the git tree and was thinking about doing the same here instead of a hardcoded fake version

Reply:
updated to determine it dynamically, e.g. azure-nvme-utils_0.1.2+git393c9ca_amd64.deb